addons/web/static/tests/helpers/utils.js

```diff
@@ -429,15 +433,56 @@ export async function mouseEnter(el, selector, coordinates) {
 
 export async function editInput(el, selector, value) {
     const input = findElement(el, selector);
-    if (
-        !["INPUT", "TEXTAREA"].includes(input.tagName) ||
-        !["text", "textarea", "email", "number", "search", "color"].includes(input.type)
-    ) {
-        throw new Error("Only inputs tag and textarea tag can be edited with editInput.");
-    }
-    input.value = value;
-    await triggerEvent(input, null, "input");
-    return triggerEvent(input, null, "change");
+    if (!(input instanceof HTMLInputElement || input instanceof HTMLTextAreaElement)) {
+        throw new Error("Only 'input' and 'textarea' elements can be edited with 'editInput'.");
+    }
+
+    let isValidValue = false;
+    switch (input.type) {
+        case "text":
+        case "textarea":
+        case "email":
+        case "search":
+        case "color": {
+            isValidValue = typeof value === "string";
+            break;
+        }
+        case "number": {
+            isValidValue = !isNaN(value);
+            break;
+        }
+        case "file": {
+            value = Array.isArray(value) ? value : [value];
+            isValidValue = value.every((f) => f instanceof File);
+            break;
+        }
+        default: {
+            throw new Error(`Type "${input.type}" not supported by 'editInput'.`);
+        }
+    }
+    if (!isValidValue) {
+        throw new Error(`Invalid value given to input`);
+    }
+
+    const eventOpts = {};
+    if (input.type === "file") {
```

Comment:
It is not used often, but great addition to the editInput helper instead of doing it manually each time👍